ruma-events/src/room/message.rs

```diff
@@ -21,57 +21,48 @@ pub use super::relationships::InReplyTo;
 pub mod feedback;
 
 use crate::MessageEvent as OuterMessageEvent;
+use crate::{from_raw_json_value, MessageDeHelper};
 
 /// This event is used when sending messages in a room.
 ///
 /// Messages are not limited to be text.
 pub type MessageEvent = OuterMessageEvent<MessageEventContent>;
 
 /// The payload for `MessageEvent`.
-#[derive(Clone, Debug, Deserialize, Serialize, MessageEventContent)]
+#[derive(Clone, Debug, MessageEventContent, Serialize)]
```

Comment:
Please don't reorder these. They are ordered the same way as our imports in most places. (std ones first, then external, then internal)